client/my-sites/plugins/main.jsx

```diff
@@ -567,12 +251,15 @@ export default React.createClass( {
 		return some( this.props.sites.getSelectedOrAllWithPlugins(), site => site && site.jetpack && site.canUpdateFiles );
 	},
 
-	updateAllPlugins() {
-		PluginsActions.removePluginsNotices( this.state.notices.completed.concat( this.state.notices.errors ) );
-		this.state.plugins.forEach( plugin => {
-			plugin.sites.forEach( site => PluginsActions.updatePlugin( site, site.plugin ) );
-		} );
-		this.recordEvent( 'Clicked Update all Plugins', true );
+	showPluginListPlaceholders( isWpCom ) {
+		let showPlaceholders = isEmpty( this.state.plugins );
```

Comment:
mainly to avoid a hard to read boolean structure in the initialization of the variable... We could use a const there, but it would look like this:
```
const selectedSite = this.props.sites.getSelectedSite();
let showPlaceholders = (
	isEmpty( this.state.plugins ) &&
	! ( selectedSite || selectedSite.jetpack === isWpCom )
);

return showPlaceholders
```
IMHO that's harder to understand than the let + if() combination (but I may be mixing "easier to read" with "more verbose"... )